.github/workflows/release-nightly.yaml

```diff
@@ -0,0 +1,68 @@
+name: Release nightly snap of upstream project
+
+on:
+  workflow_dispatch:  # Allow manual run
+  schedule:
+    - cron: "0 7 * * *"  # Every day at 07:00 UTC
+
+permissions:
+  contents: write
+  packages: write
+  issues: write
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Snapcraft in background
+        run: sudo snap install --no-wait --classic snapcraft
```

Comment:
maybe use https://github.com/snapcore/action-build instead?

Reply:
Unfortunately for the nightly build there's a different run process that doesn't work with `action-build`.